Pinta/Actions/File/SaveDocumentImplementationAction.cs

```diff
@@ -231,7 +231,8 @@ private bool SaveFile (Document document, string file, FormatDescriptor format,
 				return false;
 			}
 
-			document.Filename = Path.GetFileName (file);
+			// Set Pathname and Filename properties which triggers the document.Renamed event
+			document.PathAndFileName = file;
```

Comment:
When I went to go ahead on an 'Unsaved Image' and 'Save As..', the tooltip still only displayed its file name and not the full path. It turns out it was because the document.Pathname wasn't being set upon saving, and the tab.Tooltip just resorted to content.ContentName instead.

So instead, I changed this line to update both values as the setter for 'PathAndFileName' updates both.